docs/source/developer_guides/lora.md

```diff
@@ -152,6 +152,42 @@ The default LoRA settings in PEFT add trainable weights to the query and value l
 config = LoraConfig(target_modules="all-linear", ...)
 ```
 
+### LoRA+ optimized LoRA
```

Comment:
As is, this is a subsection of `## Initialization`. Instead, let's create a new section `## Optimizers` below and make this subsection of that section.